tests/test_runner

```diff
@@ -129,8 +129,10 @@ EOF
     shell_name=''
     shell_present=${FALSE}
     case ${shell} in
-      ash)
+        ash)
+        # shellcheck disable=SC2230
```

Comment:
I agree with the shellcheck wiki page about SC2230: https://github.com/koalaman/shellcheck/wiki/SC2230

`command -v` always works, whereas `which` doesn't always work, so it seems like a strict upgrade.